src/lib/server-utilities.js

```diff
@@ -1,9 +1,40 @@
 import fs from 'fs';
+
+// md5Hash packages
 import crypto from 'crypto';
 
+// compressJPEG packages
+import imagemin from 'imagemin';
+import imageminJpegRecompress from 'imagemin-jpeg-recompress';
+
 export function md5Hash(file) {
   const hashInstance = crypto.createHash('md5');
   // readFileSync in the syncronous version of readFile
   const fileContents = fs.readFileSync(file, 'utf8');
   return hashInstance.update(fileContents).digest('hex');
 }
+
+
+/*
+ *  Compresses JPEG image passed as file path and overwrite uncompressed file
```

Comment:
Nice comments, was reading from bottom up so was actually just about to comment all kinds of stuff about whether it compresses in place and whether it's synchronous :P. But you'd already answered my questions!